python-enum/comparison.py

```diff
@@ -0,0 +1,17 @@
+from enum import Enum
+
+
+class AtlanticAveSemaphore(Enum):
+    RED = 1
+    YELLOW = 2
+    GREEN = 3
+    PEDESTRIAN_RED = 1
+    PEDESTRIAN_GREEN = 3
+
+
+class EighthAveSemaphore(Enum):
+    RED = 1
+    YELLOW = 2
+    GREEN = 3
+    PEDESTRIAN_RED = 1
+    PEDESTRIAN_GREEN = 3
```

Comment:
:wrench: **Technical Recommendation**

Would it also make sense to include the comparison tests here?

```python
red = AtlanticAveSemaphore.RED
print(f"{red is AtlanticAveSemaphore.RED = }")
print(f"{red is not AtlanticAveSemaphore.RED = }")

yellow = AtlanticAveSemaphore.YELLOW
print(f"{yellow is red = }")
print(f"{yellow is not red = }")

pedestrian_red = AtlanticAveSemaphore.PEDESTRIAN_RED
print(f"{red is pedestrian_red = }")
```

When you use f-strings with the equals sign, then the output will look the following:

```
red is AtlanticAveSemaphore.RED = True
red is not AtlanticAveSemaphore.RED = False
yellow is red = False
yellow is not red = True
red is pedestrian_red = True
```